features/frameworks/greentea-client/source/greentea_metrics.cpp

```diff
@@ -65,8 +69,21 @@ void greentea_metrics_report()
     send_stack_info();
     Thread::attach_terminate_hook(NULL);
 #endif
+#if defined(MBED_CPU_STATS_ENABLED)
+    send_CPU_info();
+#endif
 }
 
+#if defined(MBED_CPU_STATS_ENABLED)
+static void send_CPU_info()
+{
+    greentea_send_kv("__cpu_info        up time", mbed_uptime());
+    greentea_send_kv("__cpu_info     sleep time", mbed_time_sleep());
+    greentea_send_kv("__cpu_info deepsleep time", mbed_time_deepsleep());
+    greentea_send_kv("__cpu_info % sleep / deep", (mbed_time_sleep() * 100) / mbed_uptime(), (mbed_time_deepsleep() * 100) / mbed_uptime());
```

Comment:
> (mbed_time_deepsleep() * 100) / mbed_uptime())

Not sure how useful this stat is, it says how much % time system spent in sleep since it was up. But will be interesting to see its result for different test application.